lib/memery.rb

```diff
@@ -31,6 +31,13 @@ def memoize(method_name, condition: nil, ttl: nil)
       define_memoized_method!(method_name, condition: condition, ttl: ttl)
     end
 
+    def memoized?(method_name)
+      return faslse unless defined?(@_memery_module)
```

Comment:
There is a typo so I guess we should test it.

Reply:
I agree. Sorry. Added.